book.rb

```diff
@@ -0,0 +1,10 @@
+class Book
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  attr_accessor :title, :author
+  attr_reader :rentals
+end
```

Comment:
- [ ] To implement the `has-many` side of the association between `Book` and `Rental`,  a method for adding rentals should be present in the `Book` class. Some like so:
```ruby
  def add_rental(person, date)
    Rental.new(date, self, person)
  end
```
Please kindly fix that.